pom.xml

```diff
@@ -7,40 +7,117 @@
     <groupId>org.monarchinitiative.biodownload</groupId>
     <artifactId>biodownload</artifactId>
     <version>1.0-SNAPSHOT</version>
-    <name>biodownload</name>
+    <name>BioDownload</name>
+    <description>BioDownload is a small utility library used at the Robinson lab to download HPO, MONDO, UBERON, GO ontologies.</description>
     <url>https://github.com/TheJacksonLaboratory/biodownload</url>
+    <inceptionYear>2021</inceptionYear>
+
+    <licenses>
```

Comment:
@belkassaby Can you please check if the license is OK?

Reply:
the licence is BSD 3 right?